MapboxDirections/MBRouteOptions.swift

```diff
@@ -387,6 +387,8 @@ open class RouteOptions: NSObject, NSSecureCoding {
      */
     open var distanceMeasurementSystem: MeasurementSystem = Locale.current.usesMetric ? .metric : .imperial
     
+    open var includeBannerInstructions = false
```

Comment:
Rename this property to `includesBannerInstructions` so it doesn’t look like a method in Objective-C.